### PR TITLE
Update AbstractFunction.schelp

### PR DESCRIPTION
**Pull request by madredeuz**
## Purpose and Motivation

Just update SC documentation.

## Types of changes

Introduce description for ampdb and dbamp

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

**Comment by brianlheim** [comment]
This seems a bit inefficient -- many of these methods are documented in AbstractFunction, Operators, SimpleNumber, and SequenceableCollection. Across those 4 files, some methods have single-line descriptions, some have examples, some have both, some have neither. I would much rather we consolidate that than add to the clutter. This is not a problem with your PR @madredeuz , just a general observation about how this documentation is organized.

**Comment by madredeuz** [comment]
It is documented in only two files, like this:

### Convert linear amplitude to decibels.

Maybe it could be a good idea to do the same in the other files, and contribute to create a more detailed documentation files.

I missed sometimes some more especific help, and I am programmer, so I can imagine for people that are new into SC could help a lot.

**Comment by brianlheim** [comment]
> Maybe it could be a good idea to do the same in the other files, and contribute to create a more detailed documentation files.

I think it would be even better to put it all in a single file, with redirection from the other files, so that we don't have to remember to update 4 separate files each time we want to make a change to documentation.

File: HelpSource/Classes/AbstractFunction.schelp [hunk 1]
```diff
@@ -121,7 +121,33 @@ a = Pseries(220, 220, inf).cpsmidi; a.asStream.nextN(12); // overtone series as
 method::midiratio
 method::ratiomidi
 method::ampdb
+Converts amp range (1.0 -> 0.0) to db range (ex. 0dB -> -90dB).
```
Comment (brianlheim) [inline HelpSource/Classes/AbstractFunction.schelp]: This implies that `0.0.ampdb == -90`, which is not true, can you please find a clearer way to explain that?

File: HelpSource/Classes/AbstractFunction.schelp [hunk 2]
```diff
@@ -121,7 +121,33 @@ a = Pseries(220, 220, inf).cpsmidi; a.asStream.nextN(12); // overtone series as
...
+code::
+(
+Window.closeAll;
```
Comment (brianlheim) [inline HelpSource/Classes/AbstractFunction.schelp]: can you please remove this line in both examples? it's not necessary and creates a risk that users might close something important. it's also not consistent with other examples throughout our documentation.

**Comment by brianlheim** [comment]
closing as stale

**Pull request closed without merging**